### ISSUE TITLE
[$15]"Login" button foreground and background colors missed WCAG 2 AA contrast ratio

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3090]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing
2. Right click > Inspect Elements 
3. Check the "Login" button

**Expected Result**
- Contrast ratio should be 4.5:1
- If you want passes WCAG 2 AA, you can change text color to #0074E8. New contrast ratio: 4.5:1

**Actual Result**
Actual Result
Foreground Color: #0681FF
Background Color: #FFFFFF
Contrast Ratio: 3.75:1
=> Therefor, "Login" button foreground and background colors missed WCAG 2 AA contrast ratio

**Device**: Desktop/Labtop
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=6b62afda%2Dc894%2D4ce2%2Da924%2D74186e1c15dc&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2Fdoremihong%2Fbug1

**Comment by hokienick** [comment]
This ticket may be solved once ticket #3113 has been resolved.  @kkartunov will check on this tomorrow and if it satisfies, he will close this ticket.

**Comment by Dara-K** [comment]
We should use the default button here with gray borders. The one with blue borders is more of a leftover from an older version of the site. Default buttons (first ones from here): https://community-app.topcoder-dev.com/examples/buttons/

**Comment by kkartunov** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup. PR against `qa-accessibility` branch.

**Comment by kkartunov** [comment]
Please, replace the button with the one Dara-K suggested.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097745 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097745 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097745 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
PR at https://github.com/topcoder-platform/community-app/pull/3127

**Comment by sumitdaga** [comment]
@veshu @crazyk07 @sushilshinde  
this is a duplicate with #2907 and its PR has already been accepted !

**Comment by sachin-maheshwari** [comment]
working fine as per wave tool in chrome.  On branch login-btn-fix with "topcoder-react-ui-kit": "^1.0.7"
<img width="1676" alt="Screenshot 2019-07-31 at 5 16 34 PM" src="https://user-images.githubusercontent.com/6467888/62209359-0a9d9f80-b3b7-11e9-98bc-10fe27a12d38.png">

**Comment by crazyk07** [comment]
Duplicate

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
